dev/conformance/runner.ts

```diff
@@ -384,12 +378,11 @@ function runTest(spec) {
     });
   };
 
-  const setTest = function(spec) {
+  const setTest = spec => {
     const overrides = {commit: commitHandler(spec)};
-
     return createInstance(overrides).then(() => {
-      const setOption = {};
-
+      // tslint:disable-next-line:no-any
+      const setOption: any = {};
```

Comment:
Can you make this `AnyDuringMigration`? I should be able to clean this up later.